it/generate-code.ps1

```diff
@@ -19,6 +20,23 @@ $scriptPath = Split-Path -Parent $MyInvocation.MyCommand.Definition
 $additionalArgumentCmd = Join-Path -Path $scriptPath -ChildPath "get-additional-arguments.ps1"
 $additionalArguments = Invoke-Expression "$additionalArgumentCmd -descriptionUrl $descriptionUrl -language $language"
 $rootPath = Join-Path -Path $scriptPath -ChildPath ".."
-$kiotaExec = Join-Path -Path $rootPath -ChildPath "publish"
-$kiotaExec = Join-Path -Path $kiotaExec -ChildPath "kiota"
-Start-Process "$kiotaExec" -ArgumentList "generate --language ${language} --openapi ${descriptionUrl}${additionalArguments}" -Wait -NoNewWindow
+
+switch ($dev)
+{
+    $true {
+        Write-Warning "Using kiota in dev mode"
+        $kiotaExec = Join-Path -Path $rootPath -ChildPath "src"
+        $kiotaExec = Join-Path -Path $kiotaExec -ChildPath "kiota"
+        $kiotaExec = Join-Path -Path $kiotaExec -ChildPath "bin"
+        $kiotaExec = Join-Path -Path $kiotaExec -ChildPath "Debug"
+        $kiotaExec = Join-Path -Path $kiotaExec -ChildPath "net7.0"
+        $kiotaExec = Join-Path -Path $kiotaExec -ChildPath "kiota"
```

Comment:
```suggestion
        $kiotaExec = Join-Path -Path $rootPath -ChildPath "src" -AdditionalChildPath "kiota", "bin", "Debug", "net7.0", "kiota"
```
(I've never understood why childpath is not an array like the dotnet API)

Reply:
oh, that's much nicer!